config/initializers/new_framework_defaults.rb

```diff
@@ -20,6 +20,3 @@
 
 # Require `belongs_to` associations by default. Previous versions had false.
 Rails.application.config.active_record.belongs_to_required_by_default = false
-
-# Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = true
```

Comment:
@johnpaulashenfelter This change was made because when doing `bundle exec rails s` the error below occurred 
This was removed in 5.2 https://guides.rubyonrails.org/5_2_release_notes.html#removals
```
/Users/matthewroth/dev/vets-gov/gibct-data-service/config/initializers/new_framework_defaults.rb:25:in `<top (required)>': undefined method `halt_callback_chains_on_return_false=' for ActiveSupport:Module (NoMethodError)
```